github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -1969,6 +1973,8 @@ presubmits:
           value: "true"
         - name: KUBEVIRT_WITH_ETC_IN_MEMORY
           value: "true"
+        - name: KUBEVIRT_WITH_ETC_CAPACITY
+          value: "1g"
```

Comment:
hmm, not capital? 1G?

Reply:
Yes I wasn't 100% sure but according to the tmpfs man pages it should be lower case  - https://www.man7.org/linux/man-pages/man5/tmpfs.5.html

Reply:
so we use the k8s unit parser to ensure correctness - https://github.com/kubevirt/kubevirtci/blob/main/cluster-provision/gocli/cmd/run.go#L305